src/UI/Component/Input/Field/Factory.php

```diff
@@ -205,4 +206,59 @@ public function dependantGroup(array $inputs);
 	 * @return    \ILIAS\UI\Component\Input\Field\Checkbox
 	 */
 	public function checkbox($label, $byline = null);
+
+
+	/**
+	 * ---
+	 * description:
+	 *   purpose: >
+	 *     A TagInput is used to choose a subset amount of tags out of a finite list
```

Comment:
Tag Inputs allow users to set more than one Tag from a long list of pre-existing Tags. The list of defined Tags is typically not displayed. The Tag Inputs allow user creating new Tags by typing them into the Input Field

Reply:
Did you consult: https://www.ilias.de/docu/goto_docu_wiki_wpage_4064_1357.html